<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataWidget</class>
 <widget class="QWidget" name="MetadataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="back">
     <property name="text">
      <string>Back to Scrobbles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QFrame" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>433</width>
        <height>566</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="trackDetails">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="HttpImageWidget" name="albumImage">
            <property name="text">
             <string notr="true">Album</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="unicorn::Label" name="trackTitle">
              <property name="text">
               <string notr="true">Track title</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="unicorn::Label" name="trackArtist">
              <property name="text">
               <string notr="true">Artist</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ScrobbleControls" name="scrobbleControls">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="unicorn::Label" name="trackAlbum">
              <property name="text">
               <string notr="true">Album</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="unicorn::StackedWidget" name="loadingStack">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="content">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="ContextLabel" name="context">
             <property name="text">
              <string notr="true">Context</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="trackTagsFrame">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="unicorn::Label" name="trackPopTags">
                <property name="text">
                 <string>Popular tags:</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="unicorn::Label" name="trackYourTags">
                <property name="text">
                 <string>Your tags:</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="artistFrame">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0,0,0,0,0">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="unicorn::Label" name="artistArtist">
                <property name="text">
                 <string notr="true">Artist</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="openExternalLinks">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="BioWidget" name="artistBio">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="openExternalLinks">
                 <bool>false</bool>
                </property>
                <property name="openLinks">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="artistBioEdit">
                <property name="text">
                 <string notr="true">Edited</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="artistStats">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="artistListeners">
                   <property name="text">
                    <string notr="true">0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="artistListenersLabel">
                   <property name="text">
                    <string notr="true">Listeners</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="artistPlays">
                   <property name="text">
                    <string notr="true">0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QLabel" name="artistUserPlays">
                   <property name="text">
                    <string notr="true">0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="artistPlaysLabel">
                   <property name="text">
                    <string notr="true">Plays</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="4">
                  <widget class="QLabel" name="artistUserPlaysLabel">
                   <property name="text">
                    <string notr="true">Plays in your library</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" rowspan="2">
                  <widget class="QFrame" name="splitter3">
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3" rowspan="2">
                  <widget class="QFrame" name="splitter4">
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="artistTagsFrame">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="unicorn::Label" name="artistPopTags">
                   <property name="text">
                    <string>Popular tags:</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::RichText</enum>
                   </property>
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="unicorn::Label" name="artistYourTags">
                   <property name="text">
                    <string>Your tags:</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::RichText</enum>
                   </property>
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="splitter">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="similarArtists">
                <property name="text">
                 <string>Similar Artists</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="similarArtistFrame">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="SimilarArtistWidget" name="artist1">
                   <property name="text">
                    <string notr="true">Artist</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>0</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="SimilarArtistWidget" name="artist2">
                   <property name="text">
                    <string notr="true">Artist</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>0</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="SimilarArtistWidget" name="artist3">
                   <property name="text">
                    <string notr="true">Artist</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>0</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="SimilarArtistWidget" name="artist4">
                   <property name="text">
                    <string notr="true">Artist</string>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="PlayableItemWidget" name="radio">
                <property name="text">
                 <string notr="true">Play Artist Radio</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="spinner">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="spinnerLabel">
             <property name="text">
              <string notr="true">TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HttpImageWidget</class>
   <extends>QLabel</extends>
   <header>lib/unicorn/widgets/HttpImageWidget.h</header>
  </customwidget>
  <customwidget>
   <class>unicorn::Label</class>
   <extends>QLabel</extends>
   <header>lib/unicorn/widgets/Label.h</header>
  </customwidget>
  <customwidget>
   <class>ScrobbleControls</class>
   <extends>QFrame</extends>
   <header>../Widgets/ScrobbleControls.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BioWidget</class>
   <extends>QTextBrowser</extends>
   <header>../Widgets/BioWidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlayableItemWidget</class>
   <extends>QPushButton</extends>
   <header>../Widgets/PlayableItemWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ContextLabel</class>
   <extends>QLabel</extends>
   <header>../Widgets/ContextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>SimilarArtistWidget</class>
   <extends>QLabel</extends>
   <header>../Widgets/SimilarArtistWidget.h</header>
  </customwidget>
  <customwidget>
   <class>unicorn::StackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>lib/unicorn/widgets/StackedWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
